code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -552,6 +584,12 @@
 	if(emagged)
 		reagents.check_and_add("beer2", volume, 5)
 
+//This is a special guitar for the emagged service borg that hits pretty hard and can still play music. Clonk.
+/obj/item/instrument/guitar/cyborg
```

Comment:
From what i remember, cyborg items that dont inherit from ```robot_modules``` can be placed just next to the items it inherits.